### PR TITLE
feat: Display component to show the value of the number to operate

### PR DESCRIPTION
**Pull request by marroq**

### CODE REVIEW

File: src/components/Display/index.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        <input type="text" value={number} onChange={() => changeNumber(number) } /> 
+    </div>
+);
+
+Display.propTypes = {
+    number: PropTypes.number.isRequired,
+    changeNumber: PropTypes.func.isRequired,
+};
+
+export default Display;
```
Comment (baquiax) [inline src/components/Display/index.jsx]: It's a common practice to left the last line of the file, blank.

File: src/components/Display/index.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = ({ number, changeNumber }) => (
+    <div>
+        <input type="text" value={number} onChange={() => changeNumber(number) } />
```
Comment (baquiax) [inline src/components/Display/index.jsx]: The display should be editable? 🤔 
If it is a display I think it should be read only

**Review by baquiax — CHANGES_REQUESTED** [review]

File: src/components/Display/index.jsx (hunk 1 above)
Comment (marroq) [reply src/components/Display/index.jsx]: Sure

File: src/components/Display/index.jsx (hunk 2 above)
Comment (marroq) [reply src/components/Display/index.jsx]: yes, of course

**Pull request merged**